src/modules/Tasklist.js

```diff
@@ -0,0 +1,239 @@
+import recyclebinimg from '../img/recyclebin.svg';
+import returnimg from '../img/return.svg';
+
+export default class Tasklist {
+  constructor() {
+    this.clickedonenter = false;
+    this.tasks = [];
+  }
+
+  retrievedata = () => {
+    if (localStorage.getItem('todolist') != null) {
+      this.tasks = JSON.parse(localStorage.getItem('todolist'));
+    }
+  };
+
+  savedata = () => {
+    localStorage.setItem('todolist', JSON.stringify(this.tasks));
+  };
+
+  #checkchange = (event) => {
+    const mytext = event.target.nextSibling;
+    const thisid = event.target.parentNode.id.substring(2);
+    if (event.target.checked) {
+      if (!mytext.classList.contains('tachado')) mytext.classList.add('tachado');
+      this.tasks[thisid].completed = true;
+    } else {
+      if (mytext.classList.contains('tachado')) mytext.classList.remove('tachado');
+      this.tasks[thisid].completed = false;
+    }
+    this.savedata();
+  };
+
+  #textinput = (event) => {
+    event.target.style.height = '15px';
+    event.target.style.height = `${event.target.scrollHeight}px`;
+  };
+
+  #textinput2 = (textarea) => {
+    textarea.style.height = '15px';
+    textarea.style.height = `${textarea.scrollHeight}px`;
+  };
```

Comment:
It is a recommendation I found by googling, to have a textarea always sized to the exact height of the text inside it. I don't know if adding a class in CSS would do the trick, since you have to first set the height to a fixed value and then set it again to the scrollHeight value. (I suppose in the middle the scrollHeight changes)